### ISSUE TITLE
PatchTST-FM in Darts

### ISSUE THREAD

**Issue by daidahao** [ibm-granite/granite-tsfm#486]
@wgifford

Congratulations on releasing PatchTST-FM, the latest frontier of TSFMs. I really enjoy your thorough analysis of design choices and training strategies for predictive accuracy in the paper.

Darts is a well-known Python library for time series forecasting and more recently, has incorporated latest TSFMs like [Chronos-2](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.chronos2_model.html#darts.models.forecasting.chronos2_model.Chronos2Model) and [TimesFM 2.5](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.timesfm2p5_model.html#darts.models.forecasting.timesfm2p5_model.TimesFM2p5Model) under sklearn-like forecasting APIs, for which I have been a key contributor.

To further boost adoption, would you be interested in contributing a PatchTST-FM reference implementation in Darts? If you like, I could also assist with the efforts and co-author a PR. To support TSFMs, Darts provides PyTorch-native and Lightning-enabled `FoundationModel` base class and can load `safetensors` weights on HuggingFace via `HuggingFaceConnector`, a perfect fit for PatchTST-FM.

Let me know what you think. Thank you again for your excellent work on PatchTST-FM.
